src/script.cr

```diff
@@ -3,7 +3,16 @@ module Shards
     class Error < Error
     end
 
-    def self.run(path, command, script_name, dependency_name)
+    def self.run(path : String, command : String, script_name : String, dependency_name : String, skip : Bool) : Nil
+      if !skip
+        Log.info { "#{script_name.capitalize} of #{dependency_name}: #{command}" }
+        self.run(path, command, script_name, dependency_name)
```

Comment:
Redundant `self`

```suggestion
        run(path, command, script_name, dependency_name)
```

Reply:
Redundant maybe, but it improves the readability of the code by removing the ambiguity of if it's a variable or method call.  Either way based on the discussion in #484 this PR is prob going to be closed anyway so it doesn't really matter.

Reply:
There's no need for it, since you can't _call_ a variable, nor there's a variable named _run_ in the context mentioned.